packages/devtools_extensions/example/README.md

```diff
@@ -1,46 +1,126 @@
 # Example DevTools Extension
 
-This is an end-to-end example of a DevTools extension, including
-- the parent `package:foo` that provides the DevTools extension to end-user
-applications (`foo/packages/foo`)
-- the end-user application (`app_that_uses_foo`) that depends on `package:foo`,
-which will trigger a load of the `package:foo` DevTools extension when debugging
-this app with DevTools
-- the `package:foo` DevTools extension (`foo/packages/foo_devtools_extension`),
-which is a Flutter web app that will be embedded in DevTools when debugging an
-app the uses `package:foo`
-
-
-This example will show you how to:
-1. Structure your package for optimal extension development and publishing
+This directory contains end-to-end examples of DevTools extensions. Each
+end-to-end example is made up of three components:
+
+1. **The Dart packages that provide a DevTools extension** to end-user
+applications
+
+    - `package:foo` from `packages_with_extensions/foo/packages/foo`, which is a
+  package that can be used in a Flutter app
+
+    - `package:dart_foo` from `packages_with_extensions/dart_foo/packages/dart_foo`,
+  which is a pure Dart package that can be used in a Dart or Flutter app
+
+    <!-- TODO(kenz): build this example. -->
+    <!-- - `package:standalone_tool` from `packages_with_extensions/dart_foo/packages/stanalone_tool`, which is a package that is strictly meant to provide a tool
+    as a DevTools extension. This is different from the other packages in that it
+    is not an extension shipped with an existing Dart package. It is a package
+    published solely to provide a DevTools extension. -->
+
+    <!-- TODO(kenz): build this example, or pull in Khan's extension. -->
+    <!-- - `package:gemini_ai_tool` from `packages_with_extensions/dart_foo/packages/gemini_ai_tool`, which is a standalone tool (like `package:standalone_tool`)
```

Comment:
@khanhnwin what do you think about including your extension in the `devtools_extensions/example` directory once it is complete?